vcpkg/contributing/maintainer-guide.md

```diff
@@ -443,6 +465,26 @@ Common options that allow you to avoid patching:
 - [CMAKE] Calls to `find_package(XYz)` in CMake scripts can be disabled via [`-DCMAKE_DISABLE_FIND_PACKAGE_XYz=ON`](https://cmake.org/cmake/help/v3.15/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html)
 - [CMAKE] Cache variables (declared as `set(VAR "value" CACHE STRING "Documentation")` or `option(VAR "Documentation" "Default Value")`) can be overridden by just passing them in on the command line as `-DVAR:STRING=Foo`. One notable exception is if the `FORCE` parameter is passed to `set()`. For more information, see the [CMake `set` documentation](https://cmake.org/cmake/help/v3.15/command/set.html)
 
+### <a name="prefer-download-patches"></a> Prefer downloading approved patches over checking them into the port
+
+If an approved or merged patch file can be obtained from upstream, ports should
+try to download them and apply them instead of having them as part of the port files.
+This process is prefered because it:
+
+- Confirms that upstream has accepted the patch changes
+- Simplifies the reviewing process by shifting the onus upstream
+- Reduces the vcpkg repository size for users that aren't using the patch
+- Avoids license conflicts with the vcpkg repository
+
+Patches should be downloaded from a stable endpoint to avoid SHA conflicts. 
+When downloading patch files from a pull request or commit from GitHub and
+GitLab the `?full_index=1` parameter should be appended to the download URL.

```

Comment:
Should there be a note that patch must only come from PRs which are *merged*? Anything else is mutable. Even closed PRs could be reopened.